_lifeofinternatpuppet/post_40.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Week 40
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 40
+---
+
+Hi everyone!
+
+This week has been quite short for me as I wrap up some tickets on the IAC Confluence Wiki Project before going on PTO from Wednesday 23rd June - Friday 25th June.
+
+Really looking forward to the getting some time off!
+
+The following tickets were closed and many pages were made live on Confluence :)
+- [Ticket 1](https://tickets.puppetlabs.com/browse/IAC-1583)
```

Comment:
Rather than naming Ticket 1 you could just name it IAC-1583, I feel like its more visible and makes more sense. Same for the rest of the tickets.

Reply:
Check the new commit message **Fixing ticket names**